### ISSUE TITLE
inline volume support

### ISSUE THREAD

**Issue by andyzhangx** [kubernetes-sigs/azuredisk-csi-driver#128]
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html
https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md

Need Generic Ephemeral Inline Volumes support which is only alpha state

**Describe alternatives you've considered**

**Additional context**

**Comment by andyzhangx** [comment]
1.16 is already supported by aks-engine, I think you could set up an aks-engine cluster to verify that

**Comment by andyzhangx** [comment]
if it's supported, then need to add one e2e test

**Comment by ZeroMagic** [comment]
The feature is not supported. We need to implement it in `func NodePublishVolume()`.

**Comment by letmaik** [comment]
Will inline volumes also support raw block volumes?

**Comment by andyzhangx** [comment]
> Will inline volumes also support raw block volumes?

we don't have plan to support inline volume in this driver. I think it's possible to support raw block device with inline volume feature.

**Comment by letmaik** [comment]
I don't understand your response. Could you rephrase that?

**Comment by andyzhangx** [comment]
first we have no plan to support inline volume in this driver in current stage since there is no strong requirement, and if we are going to support inline volume feature in the future, raw block device should be possible with inline volume.

**Comment by letmaik** [comment]
Thanks for confirming. I was confused because there was a PR for this already https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/255.

**Comment by andyzhangx** [comment]
there is more general inline volume support, don't need CSI driver change: https://github.com/kubernetes/enhancements/pull/1701
